exercises/prime-factors/src/test/java/PrimeFactorsCalculatorTest.java

```diff
@@ -1,42 +1,60 @@
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
 
 import java.util.Arrays;
-import java.util.Collection;
 import java.util.Collections;
-import java.util.List;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Parameterized.class)
 public class PrimeFactorsCalculatorTest {
 
-    private long input;
-    private List<Long> expectedOutput;
+    private PrimeFactorsCalculator primeFactorsCalculator;
 
-    @Parameters(name="Prime factors of {0} = {1}")
-    public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][]{
-                {1L, Collections.emptyList()},
-                {2L, Collections.singletonList(2L)},
-                {9L, Arrays.asList(3L, 3L)},
-                {8L, Arrays.asList(2L, 2L, 2L)},
-                {12L, Arrays.asList(2L, 2L, 3L)},
-                {901255L, Arrays.asList(5L, 17L, 23L, 461L)},
-                {93819012551L, Arrays.asList(11L, 9539L, 894119L)}
-        });
+    @Before
+    public void setUp() {
+        primeFactorsCalculator = new PrimeFactorsCalculator();
     }
 
-    public PrimeFactorsCalculatorTest(long input, List<Long> expectedOutput) {
-        this.input = input;
-        this.expectedOutput = expectedOutput;
+    @Test
+    public void testEmptyListResult() {
+        assertEquals(Collections.emptyList(), primeFactorsCalculator.calculatePrimeFactorsOf(1L));
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testSingleTonListResult() {
```

Comment:
The [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/prime-factors/canonical-data.json) has some good suggestions for tests names (e.g. `testPrimeNumber` for this one) which is might be good to use? We don't have to use the same test names as the canonical data but in this case they explain why these numbers have been chosen which might be useful 🙂

Reply:
Oh I didn't notice that! Thanks for the advice!

Reply:
No problem! 🙂